### PR TITLE
Add script lwaftrctl

### PR DESCRIPTION
**Pull request by dpino**
This script eases the process of running the lwaftr from within a VM. To run lwaftr virtualized is necessary to go through the following steps:

1) Bring up two NICs from the host using snabbnfv (vhost-user). 
2) Start VM with QEMU.
3) Run lwaftr from within VM.

The lwaftr is run inside the VM using the virtio_net driver.

Once the lwaftr is running, it should be possible to send packets to and from the guest. Run the "snabb-lwaftr transient" from within the host for testing purposes.

The script exposes the following commands to ease this process:

1) lwaftrctl snabbnfv start
2) lwaftrctl vm start
3) lwaftrctl lwaftr start

A file lwaftrctl.conf, containing all the necessary parameters, must exist within the same folder the lwaftrctl command is run. See lwaftrctl.conf.example.

It is also necessary to create user VM_USER in the VM. This user must be in the list of sudoers. It is necessary to create a script "run_lwaftr.sh" in VM_USER home. See run_lwaftr.sh.example.

### CODE REVIEW

**Comment by dpino** [comment]
Snippet of lwaftr virtualized performance (high peak):

```
42.999192,1.818182,7.999999,1.810595,7.387226,1.818182,7.999999,1.810278,8.544511
43.999196,1.818182,8.000000,1.809456,7.382580,1.818182,8.000000,1.808330,8.535317
44.999192,1.818182,7.999999,1.811041,7.389046,1.818182,7.999999,1.810099,8.543666
45.999191,2.045455,9.000001,2.003559,8.174520,2.045455,9.000001,1.997284,9.427180
46.999200,2.045454,8.999999,2.029920,8.282075,2.045454,8.999999,2.024419,9.555260
47.999195,2.045455,9.000000,2.026917,8.269823,2.045455,9.000000,2.021996,9.543823
48.999192,2.045455,9.000001,2.028222,8.275145,2.045455,9.000001,2.024683,9.556502
49.999193,2.045454,8.999998,2.026676,8.268839,2.045454,8.999998,2.022252,9.545030
50.999190,2.178563,9.585678,2.031885,8.290090,2.178577,9.585740,2.023756,9.552127
51.999195,2.177662,9.581715,2.000570,8.162327,2.177664,9.581724,1.994811,9.415510
52.999195,2.177664,9.581721,2.032380,8.292110,2.177664,9.581721,2.023313,9.550037
```

File: src/program/snabbnfv/nfvconfig.lua [hunk 1]
```diff
@@ -23,27 +23,29 @@ function load (file, pciaddr, sockpath)
...
    local c = config.new()
    for _,t in ipairs(ports) do
-      local vlan, mac_address = t.vlan, t.mac_address
+      local vlan, mac_address, vmdq = t.vlan, t.mac_address, t.vmdq
+      local disable_mrg_rxbuf = t.disable_mergeable_rx_buffer
       local name = port_name(t)
       local NIC = name.."_NIC"
       local Virtio = name.."_Virtio"
-      local vmdq = true
+      if vmdq == nil then vmdq = true end
```
Comment (dpino) [inline src/program/snabbnfv/nfvconfig.lua]: I'm not sure about this change. It's necessary that vmdq is false. Right now in master vmdq is only false if `mac_address` is not set in a `ports.cfg` file. Would it be possible to set a `mac_address` and disable `vmdq`? cc @lukego

Comment (lukego) [inline src/program/snabbnfv/nfvconfig.lua]: I don't think that will work. VMDq is the hardware feature that received packets to be dispatched across different queues based on their destination (DMAC w/ optional VLAN). I think it only makes sense to disable VMDq if you have one VM on the port that is supposed to receive all traffic. Having said that, the `intel10g` module API is not a 1:1 mapping onto the hardware so it is possible that the `vmdq=true` parameter has a more peculiar meaning that this.

**Comment by wingo** [comment]
You will need to rebase this onto lwaftr_starfruit.  Be sure to not pull in extraneous commits.  Sorry for the noise and thanks!

**Pull request closed without merging**